modules/indexing/src/main/java/org/apache/ignite/internal/mxbean/SqlQueryMXBean.java

```diff
@@ -178,6 +178,14 @@
         "heavy analytics query.")
     String getSqlQueryMemoryQuota();
 
+    /**
+     * Gets the amount of memory available for sql queries.
```

Comment:
Let's clarify this a bit

```suggestion
     * Gets the amount of memory available for SQL queries. This is equal to the {@link #getSqlQueryMemoryQuotaBytes} minus the amount of memory currently occupied by the queries.
```